appium/tests/data/index.ts

```diff
@@ -53,6 +53,15 @@ export const CommonData = {
   longEmail: {
     message: '\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\nText'
   },
+  updateUser: {
+    email: 'updating.key@example.test',
+    oldSignatureDate: 'Jan 12, 2022 at 11:16:58 AM',//time for CI runs
+    newSignatureDate:  'Jan 12, 2022 at 11:27:20 AM',//time for CI runs
+    // oldSignatureDate: 'Jan 12, 2022 at 9:16:58 PM', //- for local run
+    // newSignatureDate:  'Jan 12, 2022 at 9:27:20 PM', // for local run
```

Comment:
That's not good. Can you make sure that tests always run the iOS app on UTC time, both locally and in CI?

Reply:
i think it is a timezone issue

Reply:
Of course it is a timezone issue. Can you set in appium runner to run the iOS simulator on UTC timezone regardless of the timezone of the host? There must be settings for it.